### PR TITLE
Additional fixes for xmp and gpu build

### PR DESCRIPTION
**Pull request by yeounoh**
Cherry-picking:
- Fix xla_multiprocessing._setup_workers (GPU) parsing error / `caeb91a09602b3208feaf5adfb67cc70eb90a85a`
- Update CUDA capability to cover K80 / `dd52b71c9c3b0f1a4a3e2b881cbfb12a2106744a`

### CODE REVIEW

File: test/pytorch_test_base.py [hunk 1]
```diff
@@ -464,7 +464,9 @@ def on_tpuvm():
 
 class XLATestBase(DeviceTypeTestBase):
   device_type = 'xla'
-  unsupported_dtypes = {torch.half, torch.complex64, torch.complex128}
+  unsupported_dtypes = {
```
Comment (yeounoh) [inline test/pytorch_test_base.py:467]: This got leaked during rebasing, would this be OK? cc @JackCaoG

Comment (JackCaoG) [reply test/pytorch_test_base.py:467]: `torch.complex32` is no longer exist on the upstream, we must not add it back in pt/xla

Comment (yeounoh) [reply test/pytorch_test_base.py:467]: I see, so keep this?
```
unsupported_dtypes = { ... torch.complex32 ... }
```

Comment (JackCaoG) [reply test/pytorch_test_base.py:467]: You can just revert the change in this file(by patching a new commit for example).

Comment (yeounoh) [reply test/pytorch_test_base.py]: Reverted the change.

**Review by JackCaoG — APPROVED** [review]

**Pull request merged**